### DISCUSSION TITLE
Option Groups Collapse

### DISCUSSION THREAD

**Discussion by gabrielzemuner** [sa-si-dev/virtual-select#181] [Q&A] [answered]
Hello!
Is there a way to collapse option groups with virtual select?
For example, I have a year/month multiselect and I want to collapse by years.. Is it possible? 

![multipleselectoptgroup](https://user-images.githubusercontent.com/91561620/189463403-88178fe1-253c-4f19-8a8d-8fcc00b346e4.png)

**Comment by sa-si-dev** [comment, marked as answer]
@gabrielzemuner this feature is not available as of now.
